packages/pn-commons/src/components/SectionHeading.tsx

```diff
@@ -22,9 +22,8 @@ const useStyles = makeStyles(() => ({
  * @returns {JSX.Element} The rendered section heading.
  */
 function SectionHeading(props: SectionHeadingProps): JSX.Element {
-  const classes = useStyles();
   return (
-    <Typography id="title-heading-section" component="h3" variant="h6" className={classes.root}>
+    <Typography id="title-heading-section" component="h3" variant="h6" sx={style.root}>
```

Comment:
put the style into sx and convert 

`function SectionHeading(props: SectionHeadingProps): JSX.Element`

into

`const SectionHeading: React.FC = (props: SectionHeadingProps) =>`